ui/diagnoseReportApp/translations/en.yaml

```diff
@@ -107,197 +107,197 @@ diagnosis:
+        tidb_txn_kv_write_size: The KV write size per transaction execution. The label is [instance].
+        tidb_load_safepoint_total_num: The total count of safe point loading. The label is [instance].
+        tidb_lock_resolver_total_num: The total count of lock resolve. The label is [instance].
+        pseudo_estimation_total_count: The total count of TiDB Optimizer using pseudo estimation. The label is [instance, type].
+        dump_feedback_total_count: The total count of operations that TiDB dumping statistics back to KV storage. The label is [instance, type].
+        store_query_feedback_total_count: The total count of TiDB store querying feedback. The label is [instance, type].
+        update_stats_total_count: The total count of TiDB updating statistics using feed back. The label is [instance].
+        balance-leader-in: balance-leader-in is the total count of Leader moving into the TiKV store. The label is [address].
+        balance-leader-out: balance-leader-out is the total count of Leader moving out of the TiKV store. The label is [address].
+        balance-region-in: balance-region-in is the total count of Regions moving into the TiKV store. The label is [address].
+        balance-region-out: balance-region-in is the total count of Regions moving into the TiKV store. The label is [address].
+        Approximate Region size: The approximate Region size. The label is [instance].
+        store size: The storage size. The label is [instance, type].
+        tikv_scheduler_keys_read: The number of keys read by a command. The label is [instance, type].
+        tikv_scheduler_keys_written: The number of keys written by a command. The label is [instance, type].
+        tikv_scheduler_scan_details_total_num: The keys scan details of each CF when executing a command. The label is [instance,req,tag].
+        tikv_scheduler_stage_total_num: The total number of scheduler states. The label is [instance,type,stage].
+        tikv_gc_keys_total_num: The total number of keys in CF affected during GC. The label is [instance,cf,tag].
+        tidb_gc_worker_action_total_num: The total count of KV storage garbage collection. The label is [instance,type].
+        tikv_worker_handled_tasks_total_num: The total number of tasks handled by worker. The label is [instance,name].
+        tikv_worker_pending_tasks_total_num: The total number of pending and running tasks of worker. The label is [instance,name].
+        tikv_futurepool_handled_tasks_total_num: The total number of tasks handled by future_pool. The label is [instance,name].
+        tikv_futurepool_pending_tasks_total_num: The total number of pending and running tasks of future_pool. The label is [instance,name].
+        tikv_snapshot_kv_count: tikv_snapshot_kv_count. The label is [instance].
+        tikv_snapshot_size: The number of KV pairs within a snapshot. The label is [instance].
+        tikv_snapshot_state_total_count: tikv_snapshot_size. The label is [instance,type].
```

Comment:
tikv_snapshot_size is not an appropriate explanation to tikv_snapshot_state_total_count.